maintenance/README.md

```diff
@@ -15,7 +15,6 @@ To set up a local development environment:
 
 To refresh each sub-project after they have been setup run one of the refresh scripts:
```

Comment:
Please update the instructions above for step 3.

Reply:
Thanks! Done.